data/sql/updates/pending_db_world/rev_1617197895526268300.sql

```diff
@@ -0,0 +1,54 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1617197895526268300');
+-- ----------------------------------The Ancient Statuette Fixes-------------------------------------------------
+
+-- Talen
+DELETE FROM `smart_scripts` WHERE `entryorguid` IN (3846,384600);
```

Comment:
missing `source_type` in `DELETE` clause

Reply:
Done.